src/main/java/command/DeleteCommand.java

```diff
@@ -0,0 +1,55 @@
+package command;
+import exception.MikeCommandExecutionException;
+import mike.TaskList;
+import task.Task;
```

Comment:
Some inconsistent ordering of import statements compared to  DeadlineCommand.java